src/esp/gfx/replay/Player.cpp

```diff
@@ -172,6 +172,14 @@ void Player::applyKeyframe(const Keyframe& keyframe) {
   }
 }
 
+void Player::pushKeyframe(const Keyframe& keyframe) {
```

Comment:
Let's avoid the keyframe copy here. Let's move it into the vector. How about `Keyframe&& keyframe` and then `keyframes_.emplace_back(std::move(keyframe))`? I'm guessing a user doesn't need to keep around their keyframe after calling this.